content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,184 @@
+   can still be ordered.
+
+:guilabel:`Expected Arrival` is automatically calculated based on the :guilabel:`Order Deadline`
+and vendor lead time. Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at
+delivery.
+
+With the :doc:`Storage Locations feature
+<../../inventory/warehouses_storage/inventory_management/use_locations>` activated,
+the :guilabel:`Deliver to` field appears, with options for the order shipment.
+
+Select the receiving warehouse address here, or select :guilabel:`Dropship` to indicate that this
+order is to be shipped directly to the end customer. When :guilabel:`Dropship` is selected, the
+:guilabel:`Dropship address` field is enabled. Contact names auto-populate here from the
+**Contacts** app.
+
+Products tab
+~~~~~~~~~~~~
+
+In the :guilabel:`Products`` tab, add the products to be ordered. Click :guilabel:`Add a product`,
+and type in the product name, or select the item from the drop-down menu.
+
+To create a new product and add it, type the new product name in the :guilabel:`Product` column,
+select the :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the
+unit price. Or, select :guilabel:`Create and edit...` to be taken to the product form for that new
+item.
+
+:guilabel:`Catalog` can also be selected to navigate to a product menu from the chosen vendor. From
+here, products can be added to the cart.
+
+.. note::
+  To make adjustments to products and prices, access the product form by clicking the
+  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
+  the :guilabel:`Product` name.
+
+Send |RFQ|
+----------
+
+Clicking :guilabel:`Send by Email` reveals a :guilabel:`Compose Email` pop-up window, with a
+:guilabel:`Purchase: Request for Quotation` template loaded, ready to send to the vendor's email
+address (configured in the **Contacts** app).
+
+After crafting the desired message, click :guilabel:`Send`. Once sent, the |RFQ| moves to the
+:guilabel:`RFQ Sent` stage.
+
+Clicking :guilabel:`Print RFQ` downloads a PDF of the |RFQ|.
+
+Confirm order
+-------------
+
+Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active |PO|.
+
+.. tip::
+   Odoo tracks communications on each order through the chatter of the |PO| form. This shows the
+   emails sent between the user and the contact, as well as any internal notes and activities.
+   Messages, notes, and activities can also be logged on chatter.
+
+Once an |RFQ| is confirmed, it creates a |PO|.
+
+On the new |PO|, the :guilabel:`Order Deadline` field changes :guilabel:`Confirmation Date`, which
+displays the date and time the user confirmed the order.
+
+Depending on the user's chosen configuration in the **Purchase** app settings, a *vendor bill* is
+created once products have been ordered or received. For more information on managing vendor bills,
+refer to :doc:`this documentation <manage>`.
```

Comment:
It could be beneficial to the reader to use the actual title of the Documentation that's being referenced here, just so they can also locate it on their own, if needed. This is just a suggestion, though. Not a requirement. Just something to consider during your revisions. 👍